mcs/class/System/Mono.Mbedtls/CertificateHelper.cs

```diff
@@ -92,6 +101,19 @@ public static void AddToChain (Mbedtls.mbedtls_x509_crt* chain, IntPtr crt, int
 			Mbedtls.unity_mbedtls_x509_crt_parse (chain, crt, (IntPtr)crtLength);
 		}
 
+		public static void AddFileToChain (Mbedtls.mbedtls_x509_crt* chain, string path)
+		{
+			if (String.IsNullOrEmpty(path))
```

Comment:
Do you need a File.Exists check too? Or perhaps check return code from `unity_mbedtls_x509_crt_parse_file` below?